src/api/dlp/BorrowerStandardPortalLegalDocumentApi.ts

```diff
@@ -11,15 +13,23 @@ export interface BorrowerStandardPortalLegalDocument {
   updatedBy?: UserShort | null;
 }
 
+export interface GetLegalDocumentsParams {
+  environment: DigitalLendingPortalEnvironment;
+}
+
 class BorrowerStandardPortalLegalDocumentApi {
   protected path = '/borrowers/standard-portals/legal-documents';
 
   constructor(
     private apiClient: IApiClient,
   ) {}
 
-  public getLegalDocuments(): Promise<BorrowerStandardPortalLegalDocument[]> {
-    return this.apiClient.makeCall(`${this.path}`);
+  public getLegalDocuments(params: GetLegalDocumentsParams): Promise<BorrowerStandardPortalLegalDocument[]> {
+    const urlSearchParams = getSearchParams({
+      environment: params?.environment || DigitalLendingPortalEnvironment.Production,
+    } as Record<string, string>);
+
+    return this.apiClient.makeCall(`${this.path}/${urlSearchParams}`);
```

Comment:
missing question mark this.apiClient.makeCall(`${this.path}?${urlSearchParams}`);